app/models/media_object.rb

```diff
@@ -272,22 +275,22 @@ def to_solr(include_child_fields: false)
       all_text_values = []
       all_text_values << solr_doc["title_tesi"]
       all_text_values << solr_doc["creator_ssim"]
-      all_text_values << solr_doc["contributor_sim"]
+      all_text_values << solr_doc["contributor_ssim"]
       all_text_values << solr_doc["unit_ssim"]
       all_text_values << solr_doc["collection_ssim"]
-      all_text_values << solr_doc["summary_ssi"]
-      all_text_values << solr_doc["publisher_sim"]
-      all_text_values << solr_doc["subject_topic_sim"]
-      all_text_values << solr_doc["subject_geographic_sim"]
-      all_text_values << solr_doc["subject_temporal_sim"]
-      all_text_values << solr_doc["genre_sim"]
-      all_text_values << solr_doc["language_sim"]
-      all_text_values << solr_doc["physical_description_sim"]
+      all_text_values << solr_doc["abstract_ssi"]
+      all_text_values << solr_doc["publisher_ssim"]
+      all_text_values << solr_doc["topical_subject_ssim"]
+      all_text_values << solr_doc["geographic_subject_ssim"]
+      all_text_values << solr_doc["temporal_subject_ssim"]
+      all_text_values << solr_doc["genre_ssim"]
+      all_text_values << solr_doc["language_ssim"]
+      all_text_values << solr_doc["physical_description_ssim"]
       all_text_values << solr_doc["series_ssim"]
       all_text_values << solr_doc["date_sim"]
       all_text_values << solr_doc["notes_sim"]
-      all_text_values << solr_doc["table_of_contents_sim"]
-      all_text_values << solr_doc["other_identifier_sim"]
+      all_text_values << solr_doc["table_of_contents_ssim"]
+      all_text_values << solr_doc["other_identifier_ssm"]
```

Comment:
I think this should go back to `sim` like `notes_sim` above so it is just the values instead of the json hashes.
```suggestion
      all_text_values << solr_doc["other_identifier_sim"]
```